zerver/openapi/zulip.yaml

```diff
@@ -909,6 +909,26 @@ paths:
           - change_all
           default: change_one
         example: change_all
+      - name: send_notification_to_old_thread
+        in: query
+        description: |
+          "Whether to send breadcrumb message in the old thread to
+           notify users where this thread was moved to."
+        schema:
+          type: boolean
+          default: true
+        allowEmptyValue: true
+        example: true
+      - name: send_notification_to_new_thread
+        in: query
+        description: |
+          "Whether to send breadcrumb message in the new thread to
+           notify users where this thread was moved from."
+        schema:
+          type: boolean
+          default: true
+        allowEmptyValue: true
```

Comment:
I don't think we want `allowEmptyValue`; what you mean is `required: false`, which I think is the default?

Reply:
https://swagger.io/docs/specification/describing-parameters/ makes clear this was wrong.

Reply:
oh, Thanks!